packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -450,6 +450,31 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
           const s = new MagicString(code)
           const rewroteMarkerStartPos = new Set() // position of the leading double quote
 
+          type FileDep = {
+            url: string
+            runtime: boolean
+          }
```

Comment:
It would nice to move this up of the file outside of this function for consistency with the other code.